ddp-workspace/projects/ddp-sdk/src/lib/services/authentication/auth0Adapter.service.ts

```diff
@@ -134,8 +134,9 @@ export class Auth0AdapterService implements OnDestroy {
             }),
             language: this.language.getCurrentLanguage()
         };
-        this.log.logToCloud(`Auth0 login modal is open: ${JSON.stringify(params)}`, { auth0Mode: Auth0Mode.LoginOnly });
-        this.showAuth0Modal(Auth0Mode.LoginOnly, params);
+        this.log.logToCloud(`Auth0 login modal is open: ${JSON.stringify(params)}`, { auth0Mode: Auth0Mode.LoginOnly })
```

Comment:
Suggest adding `takeUntil(this.ngUnsubscribe)` here and for other subscriptions.

Reply:
Went around and added .pipe(take(1)) everywhere, which I think now that I think about, is probably a better way to do it (everytime we subscribe we unsubscribe immediately).
cbb033159c64faf427db8e73c65dc00401a09d9a